src/hardware/power.hpp

```diff
@@ -4,14 +4,14 @@
 
 namespace nest {
 namespace mc {
-namespace util {
+namespace hw {
 
 // Energy in Joules (J)
 using energy_size_type = std::uint64_t;
 
 // Returns negative value if unable to read energy
```

Comment:
Just noticed comment (I know this is older code): 'negative value' doesn't make sense with `uint64_t` return type.